style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+    color: green;
```

Comment:
- [ ] You have a linter error.  line 2 should only be indented by spaces. kindly fix this.
![stylint](https://user-images.githubusercontent.com/99656687/226588008-7eae520f-77d3-476a-b2f9-9069ca5dc25a.png)